project/config/afbini/config/block.block.afbini_theme_views_block__content_by_site_topic_site_topic_news.yml

```diff
@@ -0,0 +1,34 @@
+uuid: 8e23c0e2-7560-4799-bf56-36b37d6c261d
+langcode: en
+status: true
+dependencies:
+  config:
+    - views.view.content_by_site_topic
+  module:
+    - block_visibility_groups
+    - system
+    - views
+  theme:
+    - afbini_theme
+id: afbini_theme_views_block__content_by_site_topic_site_topic_news
+theme: afbini_theme
+region: related
+weight: -10
+provider: null
+plugin: 'views_block:content_by_site_topic-site_topic_news'
+settings:
+  id: 'views_block:content_by_site_topic-site_topic_news'
+  label: ''
+  label_display: visible
+  provider: views
+  views_label: ''
+  items_per_page: '2'
+visibility:
+  condition_group:
+    id: condition_group
+    negate: false
+    block_visibility_group: ''
+  request_path:
+    id: request_path
+    negate: false
+    pages: "/area-of-expertise/animal-health-and-welfare\r\n/area-of-expertise/sustainable-livestock\r\n/area-of-expertise/grassland-and-plant-science\r\n/area-of-expertise/environmental-protection\r\n/area-of-expertise/food-safety\r\n/area-of-expertise/fisheries-and-aquatic-ecosystems\r\n/area-of-expertise/economics"
```

Comment:
Could this be condensed to /area-of-expertise/* to allow for changes to the taxonomy?